.github/README.md

```diff
@@ -1,6 +1,6 @@
 # Q-CTRL DevOps Challenge
 
-The Q-CTRL DevOps Challenge is a way for applicants to roles within the Q-CTRL DevOps Engineering team to demonstrate their overall approach in creating solutions from a variety of requirements.
+The Q-CTRL DevOps Challenge is a way for applicants to apply for roles within the Q-CTRL DevOps Engineering team and demonstrate their overall approach to creating solutions from a variety of requirements.
```

Comment:
```suggestion
The Q-CTRL DevOps Challenge is a way for applicants applying for a role within the Q-CTRL DevOps Engineering team to demonstrate their overall approach to creating solutions from a variety of requirements.
```
Thoughts? I think it's not a way to 'apply for' roles, that's through the job portal etc.

Reply:
Yep, I like this much better!